recipes/mpas-analysis/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "MPAS-Analysis" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/MPAS-Dev/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9a9c99c8a6c510d2f552a88744095ae31c67cb07f7b519d9ba6f6543f4584406
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+    noarch: python
+    entry_points:
+        - mpas_analysis = mpas_analysis.__main__:main
```

Comment:
Before I forget, there is a second entry_point in the `setup.py` called `download_analysis_data`.

Reply:
There sure is, and it's missing from my own conda recipe, meaning the packages on my channel are broken. Thank you for noticing that.